acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -48,5 +48,15 @@
 install_packages_on(master, MASTER_PACKAGES)
 install_packages_on(agents, AGENT_PACKAGES)
 
+agents.each do |agent|
+  if agent['platform'] =~ /windows/
+    arch = agent[:ruby_arch] || 'x86'
+    base_url = "http://builds.puppetlabs.lan/puppet/#{ENV['SHA']}/artifacts/windows"
+    filename = "puppet-agent-#{ENV['VERSION']}-#{arch}.msi"
```

Comment:
The way it is done now, if you pass in a name, it will take it wholesale without attempting to change it. https://github.com/puppetlabs/puppet_for_the_win/blob/master/tasks/windows/windows.rake#L116